grunttasks/l10n-htmllint.js

```diff
@@ -34,10 +34,12 @@ module.exports = function (grunt) {
         }
       }
       if (content !== '') {
-        grunt.file.write(path.join('<%= yeoman.tmp %>', pathname[1], pathname[2], pathname[3].replace('json', 'html')), content);
+        // Changing back to .tmp as yeoman.tmp seems to not work,
```

Comment:
This comment is not useful (has context that will not be relevant later on), we can remove the 2 comment lines.